src/services/datahub-client.test.ts

```diff
@@ -33,10 +33,12 @@ const fullTrafficSourceResponse = {
       {
         sourceName: 'Email',
         quantity: 2000,
```

Comment:
Perhaps in this PR you could also add another commit that transforms this quantity into `withoutEmail` field?

Reply:
dont use other field. The idea is show in the view `quantity - withEmail` like in page ranking report